doc/releases/index.rst

```diff
@@ -55,7 +55,10 @@ receive bug fixes or backports).
 Release timeline
 ----------------
 
-.. ceph_timeline:: releases.yml pacific octopus nautilus
+.. ceph_timeline:: releases.yml quincy pacific octopus nautilus
```

Comment:
One of these arguments should be removed (at least, according to the warning). Is it nautilus that needs to be removed since it's the oldest?

Reply:
I've fixed this in my most recent (squashed) commit.

https://github.com/ceph/ceph/pull/45674